cli/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ _Released 02/14/2023 (PENDING)_
 
 - Fixed an issue with the Cloud project selection modal not showing the correct prompts. Fixes [#25520](https://github.com/cypress-io/cypress/issues/25520).
 - Fixed an issue in middleware where error-handling code could itself generate an error and fail to report the original issue. Fixes [#22825](https://github.com/cypress-io/cypress/issues/22825).
+- Fixed an issue in the Debug page where in-progress runs displayed a different count of specs locally compared to the count in Cypress Cloud. Fixes [#25647](https://github.com/cypress-io/cypress/issues/25647).
```

Comment:
```suggestion
- Fixed an issue that could cause the Debug page to display a different number of specs for in-progress runs than shown in Cypress Cloud. Fixes [#25647](https://github.com/cypress-io/cypress/issues/25647).
```

Reply:
Just trying to rework to remove the reference to "locally". Since this is the App changelog everything in here is implicitly "local"

Reply:
Yeah this is better.